### DISCUSSION TITLE
SectionLink method problem with Unicode characters

### DISCUSSION THREAD

**Discussion by elvinsadiq** [QuestPDF/QuestPDF#952] [Q&A]
SectionLink method doesn't work if Unicode characters are included such as "ə ğ ö ü ı"

**Comment by MarcinZiabek** [comment]
Thank you for sharing this observation 😄  Would you please elaborate a bit more about your use-case?

At this moment, I am unsure if it is just a bug in QuestPDF or an actual limitation of the Skia library or the PDF format. I recommend always using ASCII to specify section names and links. 

I will put this item on the roadmap to investigate the possible changes to support Unicode characters fully.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
